core/src/main/scala/com/nulabinc/backlog/migration/common/domain/BacklogStatus.scala

```diff
@@ -0,0 +1,55 @@
+package com.nulabinc.backlog.migration.common.domain
+
+import com.nulabinc.backlog4j.Status
+
+case class BacklogStatuses(private val values: Seq[BacklogStatus]) {
```

Comment:
BacklogStatuses provides domain logic for the status array, not just a value wrapper class